includes/class-wcpdf-admin.php

```diff
@@ -416,11 +416,11 @@ public function data_input_box_content ( $post ) {
 				<!-- Editable -->
 				<div class="editable">
 					<p class="form-field _wcpdf_invoice_number_field ">
-						<label for="_wcpdf_invoice_number"><?php _e( 'Invoice Number (unformatted!)', 'woocommerce-pdf-invoices-packing-slips' ); ?>:</label>
+						<label for="wcpdf_invoice_number"><?php _e( 'Invoice Number (unformatted!)', 'woocommerce-pdf-invoices-packing-slips' ); ?>:</label>
```

Comment:
in pro we actually prefix everything with an underscore already... so maybe follow suit here in free too